pkgs/development/python-modules/uproot/default.nix

```diff
@@ -0,0 +1,27 @@
+{lib, fetchFromGitHub, buildPythonPackage, numpy }:
+
+buildPythonPackage rec {
+  name = "uproot-${version}";
```

Comment:
The `name` attribute is added by `buildPython*` when `pname` is added, and should therefore be removed.